packages/driver/src/cypress/cy.ts

```diff
@@ -121,6 +120,14 @@ const setTopOnError = function (Cypress, cy: $Cy) {
   top.__alreadySetErrorHandlers__ = true
 }
 
+const ensureRunnable = (cy, cmd) => {
```

Comment:
ensureRunnable is only used in this one file, so I moved it here to be close to where it's used.